emcc

```diff
@@ -1385,7 +1386,7 @@ try:
 
   if memory_init_file:
     if shared.Settings.USE_TYPED_ARRAYS != 2:
-      if type(memory_init_file) == int: logging.warning('memory init file requires typed arrays mode 2')
```

Comment:
Dropping this typ echeck means that I'll now have greater chances of hitting this message. I guess I should introduce some other variable to distinguish a user-requested setting from an automatically chosen one. Or should I use extra values for that single variable?